pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -50,7 +50,7 @@ stdenv.mkDerivation rec {
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
 
-  nativeBuildInputs = [ makeWrapper removeReferencesTo pkg-config flex bison meson ninja perl python3 python3.pkgs.sphinx python3.pkgs.sphinx-rtd-theme ]
+  nativeBuildInputs = [ makeWrapper removeReferencesTo pkg-config flex bison meson ninja perl python3 buildPackages.python3.pkgs.sphinx buildPackages.python3.pkgs.sphinx-rtd-theme ]
```

Comment:
It seems strange that we'd need this.  Do you know why splicing doesn't take care of it for us?

Reply:
Apparently only the argument `python3` is spliced, not the 'nested' derivations